askama/src/filters/builtin.rs

```diff
@@ -455,6 +455,63 @@ impl<S: FastWritable, P: FastWritable> FastWritable for Pluralize<S, P> {
     }
 }
 
+/// Returns an iterator without filtered out values.
+///
+/// ```
+/// # use askama::Template;
+/// #[derive(Template)]
+/// #[template(
+///       ext = "html",
+///       source = r#"{% for elem in strs|reject("a") %}{{ elem }},{% endfor %}"#,
+/// )]
+/// struct Example<'a> {
+///     strs: Vec<&'a str>,
+/// }
+///
+/// assert_eq!(
+///     Example { strs: vec!["a", "b", "c"] }.to_string(),
+///     "b,c,"
+/// );
+/// ```
+pub fn reject<T: PartialEq>(
+    it: impl IntoIterator<Item = T>,
+    filter: T,
```

Comment:
Maybe better `filter: &T`, with an additional `&(…)` in the generated code. This way (I think) the type-coercion might work better if the input is already an `&T`.

Reply:
Anyway, added a second one.

Reply:
No, that's not what I meant, but I confused myself with my comment. Let me think for a second or 86400. :)